### PR TITLE
ADD: Change Android soft keys bar along with Theme

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ktyttdrgpeu2mrcup16efquuvw

[download apk](http://filestorage.bluewallet.io:1488/1605562576.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1y1v6dktk54bwkt11wc1nq0p08

[download apk](http://filestorage.bluewallet.io:1488/1605580562.apk)

**Review by ncoelho — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -134,6 +134,7 @@
     "react-native-linear-gradient": "2.5.6",
     "react-native-localize": "1.4.2",
     "react-native-modal": "11.5.6",
+    "react-native-navigation-bar-color": "git+https://github.com/BlueWallet/react-native-navigation-bar-color.git#34e44b8f44e442133de9d35c35f2679d40982804",
```
Comment (Overtorment) [inline package.json:137]: i hate it when we add deps

Comment (Overtorment) [reply package.json:137]: is there a way  to NOT add deps?

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**